src/task/src/repos/gitHubReposInvoker.ts

```diff
@@ -108,8 +108,8 @@ export default class GitHubReposInvoker extends BaseReposInvoker {
 
     this.initialize();
 
-    let pullRequestComments: GetIssueCommentsResponse | undefined;
-    let fileComments: GetReviewCommentsResponse | undefined;
+    let pullRequestComments: GetIssueCommentsResponse | null = null;
+    let fileComments: GetReviewCommentsResponse | null = null;
```

Comment:
There are a lot of these "null = null" changes throughout so I'm going to stop commenting on each of them here.
What is the intent with these?

Reply:
See earlier answer